.scrutinizer.yml

```diff
@@ -17,6 +17,8 @@ filter:
     - 'vendor/'
 
 build:
+  environment:
+    php: 8.3.0
```

Comment:
The CI only runs without fail if I add this to the scrutinizer config.

Else, Scrutinizer runs with php7.3 and composer v1 for some reason.

Reply:
Unfortunately, specifying that PHP version with the low patch number is *exactly* the solution IIR. If specifying `8.3` does not work [as advertised in the docs](https://scrutinizer-ci.com/docs/guides/php/continuous-integration-deployment#php-version) you might try a more recent one (8.3.10 or maybe 8.3.9) to see if that works. Otherwise let's hope that the bugs in 8.3.0 won't affect Scrutinizer ...